README.md

```diff
@@ -1 +1,105 @@
-# MobilliumGitGuidelines
\ No newline at end of file
+
+
+# Mobillium Git Guidelines
+
+### Table of Contents
+
+1. [Master & Develop Branches](#master-develop)
+1. [Feature & Epic Branches](#feature-epic)
+1. [Release & Bugfix Branches](#release-bugfix)
+1. [Hotfix Branches](#hotfix)
+1. [Summary](#summary)
+1. [References](#references)
+
+
+<a name="master-develop"></a>
+## Master & Develop Branches
+
+![](images/master-develop.svg)
+
+We have two main branches. These are `master` and `develop`.
+* The `master` branch store the latest release codebase.
+* The `develop` branch store the updated codebase.
+
+
+
+<a name="feature-epic"></a>
+## Feature & Epic Branches
+
+![](images/feature-epic.svg)
+
+### Small Tasks
+* Each new `small` `feature` should be created from `develop`.
+* Branch name should be start with `feature/` and end with task name.
+* After the tests are completed and approved, they should be merged back to related `develop` branch.
+
+### Large Tasks
+* Each new `large` `feature` should be created from `develop`.
+* Branch name should be start with `epic/` and end with task name.
+* After the tests are completed and approved, they should be merged back to related `develop` branch.
+
+
+* Sub-tasks of Large Tasks are should be created from `epic`.
+* Branch name should be start with `feature/` and end with task name.
+* After the tests are completed and approved, they should be merged back to related `epic` branch.
+
+
+
+<a name="release-bugfix"></a>
+## Release & Bugfix Branches
+
+![](images/release-bugfix.svg)
+
+> * ***Only `bugfix` can be made on the `release` branch.***
+> * ***No new feature can be added to after branch created.***
+
+### Release Branches
+* Each new `release` should be created from `develop`.
+* Branch name should be start with `release/` and end with version number(e.g. `release/1.0.1`).
+* After the new version is released, they should be merged to `master` and `develop` branches, and `master` should be tagged with an updated version numbers.
+
+
+### Bugfix Branches
+* Each new `bugfix` should be created from `release`.
+* Branch name should be start with `bugfix/` and end with task name.
+* After the tests are completed and approved, they should be merged back to related `release` branch.
+
+
+<a name="hotfix"></a>
+## Hotfix Branches
+
+![](images/hotfix.svg)
+
+
+* The `hotfix` branches are used for quick bugfix on production releases.
+* Each new `hotfix` should be created from `release`.
+* Branch name should be start with `hotfix/` and end with task name.
```

Comment:
If branch end with version number like release branches we can trigger the CI system with this version number. For my opinion there is no need for task name instead we can use version number

Reply:
fix typo